vmicore/src/vmi/Event.h

```diff
@@ -1,35 +1,43 @@
 #ifndef VMICORE_EVENT_H
 #define VMICORE_EVENT_H
 
-#include "../io/ILogging.h"
-#include "../io/grpc/GRPCLogger.h"
-#include "LibvmiInterface.h"
 #include <cstdint>
 #include <libvmi/events.h>
-#include <memory>
-#include <utility>
+#include <vmicore/vmi/events/IInterruptEvent.h>
 
 namespace VmiCore
 {
-    class Event
+    class Event : public IInterruptEvent
```

Comment:
What happends for other types of events ? (e.g. memory / register)

Reply:
Support for those is not implemented yet. We should create a followup PR as soon as we need it.